recipe/meta.yaml

```diff
@@ -39,15 +39,15 @@ outputs:
         - git
         - pkg-config
       host:
-        - abseil-cpp =20210324.2
+        - abseil-cpp
         - libprotobuf
         - zlib 1.2.11
         # coin or deps
-        - coin-or-utils 2.11.4
-        - coin-or-osi 0.108.6
-        - coin-or-clp 1.17.4
-        - coin-or-cgl 0.60.3
-        - coin-or-cbc 2.10.5
+        - coin-or-utils
```

Comment:
@wolfv Not sure how to add an upper bound of 1.17.4 here?

Reply:
we could either keep an upper bound in the meta.yaml or add a custom `conda_build_config.yaml` file.